src/sentry/metrics/logging.py

```diff
@@ -18,7 +18,7 @@ def incr(self, key, instance=None, tags=None, amount=1, sample_rate=1):
 
     def timing(self, key, value, instance=None, tags=None, sample_rate=1):
         logger.debug(
-            '%r: %g ms', key, value, extra={
+            '%r: %g ms', key, value * 1000, extra={
```

Comment:
This is unrelated, but this value was just wrong.

Reply:
😅